docs/source/cli/graph-construction/single-machine-gconstruct.rst

```diff
@@ -3,181 +3,325 @@
+                        "feature_name": "embed",
+                        "out_dtype": 'float16',
+                    },
+                ],
+
+            },
         ],
         "edges": [
             {
-                "source_id_col":    "src_paper_id",
-                "dest_id_col":      "dest_paper_id",
-                "relation":         ["paper", "cite", "paer"],
+                "source_id_col":    "nid",
+                "dest_id_col":      "domain",
+                "relation":         ["paper", "has", "subject"],
                 "format":           {"name": "parquet"},
-                "files":            ["/tmp/edge_feat.parquet"],
-                "features":         [
+                "files":            ["paper_has_subject_edges.parquet"],
+                "labels":       [
                     {
-                        "feature_col":  ["citation_time"],
-                        "feature_name": "feat",
+                        "label_col": "cnt",
+                        "task_type": "regression",
+                        "custom_split_filenames": {
+                                            "train": "train_edges.json",
+                                            "valid": "val_edges.json",
+                                            },
                     },
-                ]
+                ],
+            },
+            {
+                "source_id_col":    "nid",
+                "dest_id_col":      "n_id",
+                "relation":         ["paper", "written-by", "author"],
+                "format":           {"name": "parquet"},
+                "files":            ["paper_written-by_author_edges.parquet"],
             }
         ]
     }
 
+.. note:: For a real runnable example, please refer to the :ref:`input JSON file <input-config>` used in the :ref:`Use Your Own Graphs Tutorial <use-own-data>`.
+
+A full argument list of the ``gconstruct.construct_graph`` command
+...................................................................
+
+* **-\-conf-file**: (**Required**) the path of the configuration JSON file.
+* **-\-num-processes**: the number of processes to process the data simulteneously. Default is 1. Increase this number can speed up data processing.
+* **-\-num-processes-for-nodes**: the number of processes to process node data simulteneously. Increase this number can speed up node data processing.
+* **-\-num-processes-for-edges**: the number of processes to process edge data simulteneously. Increase this number can speed up edge data processing.
+* **-\-output-dir**: (**Required**) the path of the output data files.
+* **-\-graph-name**: (**Required**) the name assigned for the graph.
+* **-\-remap-node-id**: boolean value to decide whether to rename node IDs or not. Adding this argument will set it to be true, otherwise false.
+* **-\-add-reverse-edges**: boolean value to decide whether to add reverse edges for the given graph. Adding this argument will set it to be true, otherwise false.
```

Comment:
Shall we highlight that usually we add this argument?

Reply:
Yes, updated as a recommendation.